### ISSUE TITLE
US anti-air datalinks too short ranged

### ISSUE THREAD

**Issue by FIM43-Redeye** [PygmalionOfCyprus/cmo-db-requests#5363]
### DB Selector

DB3K

### Affected DBID(s)

Comms (????)

### Summary of Changes

All the Aegis datalinks I can find are too short-ranged to manage a long-range SM-3 or SM-6 engagement at only 160nm. I can't understand why the range is so low. The command datalink on the E-2 is also too short-ranged, as well as the AIM-120 datalink, but that's theory.

The DLs used to be 1400nm, and the one-way missile datalink still is. That seems much more sensible to me.

### Sources

Previous DBs

**Comment by RickDNiu** [comment]
Not really. SPY-1 can't support the full range of SM-3 Blk IIA even engaging on remote with LREW/satellite data over C2BMC. 
https://github.com/PygmalionOfCyprus/cmo-db-requests/issues/3428#issuecomment-2354782514
https://github.com/PygmalionOfCyprus/cmo-db-requests/issues/5282
https://github.com/PygmalionOfCyprus/cmo-db-requests/issues/5063
